README.dev.md

```diff
@@ -0,0 +1,40 @@
+Welcome to ppx_import, and thanks for willing to contribute to its
+development.
+
+## How to release a new version of ppx_import
+
+**Short version:** Run `git tag -s` + `dune-release`.
+
+The preferred release workflow is to use `dune-release`. In this
+workflow, the first (and crucial) step is to tag the release and push
+it to the main repository.
+
+We recommend you do this manually, using `git tag -s`. Note the tag
+must be "signed", and you should add the changelog to the tag commit
+message
```

Comment:
What is "must" exactly? I don't use signed commits or tags, for example, the UI of them is atrocious.

Reply:
Indeed using `git tag -a` may suffice, let me check. `dune-release` / `topkg` won't consider regular tags a releases I understand.

Reply:
Indeed, the version is guessed by using `git describe` so annotated tags will work, amending.